test/parallel/test-net-connect-local-error.js

```diff
@@ -3,25 +3,40 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+function getUnassignedPort(cb) {
+  const server = net.createServer();
+  server.listen({port: 0}, common.mustCall(() => {
+    // When the server is closed this port will no longer be assigned
+    const unassignedPort = server.address().port;
+    server.close(common.mustCall(() => {
```

Comment:
> IMHO port + 1 will always be flaky even in sequential.

I think that's wrong, unless you're just arguing that some other process can always use that port. But I don't think we generally concern ourselves with that. I'm with @santigimeno: Moving it to `sequential` seems like the simpler and better option.